### ISSUE TITLE
Will it improve performance if I apply other stable diffusion model which is finetuned on certain content images?

### ISSUE THREAD

**Issue by eeyrw** [XPixelGroup/DiffBIR#62]
As title said.

**Comment by 0x3f3f3f3fun** [comment]
I think Stable Diffusion v2 may not be suitable for some special domains, such as medical image. The reason is that stable diffusion is not able to generate those domain-specific images. So if you use other diffusion models finetuned on certain content images, I guess🤔 the performance will be improved.

**Comment by eeyrw** [comment]
Yes. The certain content images refer general images with specific content such as human face rather than domain specific images. Is replacing weight of UNet by existing model and not finetuning LAControNet  (plug-and-play) OKay?
